src/VerifierAlg/ES256KVerifierAlg.ts

```diff
@@ -0,0 +1,91 @@
+import { ec as EC, SignatureInput } from 'elliptic'
+import { sha256, toEthereumAddress } from '../Digest'
+import type { VerificationMethod } from 'did-resolver'
+import { bytesToHex, EcdsaSignature } from '../util'
+import { verifyBlockchainAccountId } from '../blockchains'
+import * as CommonVerifierAlg from './CommonVerifierAlg'
+
+const secp256k1 = new EC('secp256k1')
+
+/*
+interface LegacyVerificationMethod extends VerificationMethod {
```

Comment:
I prefer this simple extension of the publicly defined type as opposed to redefining the interfaces all together.
It seems clearer about what the legacy bits are.